packages/app/public/index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" dir="ltr">
```

Comment:
Should we be able to set this from e.g. `<UI>` or do we have to do this for each app?

Reply:
Setting this in `public/index.html` would help us let go of any `React` dependency where we keep a reference to the `<UI /> Component` itself.

It also gives us generic way to update `dir={ltr,rtl}` in all the apps.

Reply:
Ok.